config/environments/test.rb

```diff
@@ -23,12 +24,12 @@
   }
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local = true
   config.action_controller.perform_caching = false
   config.cache_store = :null_store
 
-  # Raise exceptions instead of rendering exception templates.
-  config.action_dispatch.show_exceptions = false
+  # Render exception templates for rescuable exceptions and raise for other exceptions.
+  config.action_dispatch.show_exceptions = :rescuable
```

Comment:
Why changed this? Can some tests be confused by this change?

Reply:
Previously when exception came, it would raise and test would fail. Now if the test doesn't check page output, it may miss an exception. So there might be false negatives.

Such will not show up as failed tests but as tests that previously would fail and not anymore.. that's my concerned but hopefully mosts tests will check output.

Reply:
Hm, for some reason I can't reproduce this deprecation in zync, but I did see it in porta: https://github.com/3scale/porta/pull/4129/commits/638140998a0c0381750529e548efb75af6910929#r2568854262

```
DEPRECATION WARNING: Setting action_dispatch.show_exceptions to false is deprecated. Set to :none instead.
```